web-admin/src/routes/[organization]/+page.svelte

```diff
@@ -13,7 +13,9 @@
   );
 
   $: if ($projs.data && $projs.data.projects?.length > 0) {
-    goto(`/${$page.params.organization}/${$projs.data.projects[0].name}`);
+    goto(
+      `/${$page.params.organization}/${$projs.data.projects[0].name}/-/redirect`
+    );
```

Comment:
Any reason not to have the redirect logic in this page?

Reply:
The redirect logic depends on calls to a project's runtime, so it's just easier to do within the scope of `/[organization]/[project]/+layout.svelte`, which passes the project's runtime to child components.